src/quic/node_quic_crypto.cc

```diff
@@ -368,7 +368,7 @@ bool CheckCertNames(
     const std::string& name,
     bool use_wildcard = true) {
 
-  if (name.length() == 0)
+  if (name.empty())
```

Comment:
Please separate out the change to the quic file into a separate commit so that the rest of the changes can be more easily backported.

Reply:
Seems like the code in question was deleted.